services/brig/Makefile

```diff
@@ -90,18 +90,18 @@ integration-list: install
 integration-%: db-migrate index-reset
 	LOG_LEVEL=Info keiretsu --run '$(EXE_IT) --pattern="$*"' --env $(KEIRETSU_ENV)
 
+
+.PHONY: fast
+fast:
```

Comment:
shouldn't this be:

```
.PHONY:
fast: .PHONY
```

Reply:
I've not seen this, I thought `.PHONY: fast` only to mean the `fast` target is always out-of-date, i.e. `make fast` will run even if there is a local file with the name `fast`. That's the explanation found on [stackoverflow](https://stackoverflow.com/questions/2145590/what-is-the-purpose-of-phony-in-a-makefile) at least.